src/main/scala/com/feedzai/cosytest/DockerComposeJavaSetup.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * The copyright of this file belongs to Feedzai. The file cannot be
+ * reproduced in whole or part, stored in a retrieval system,
+ * transmitted in any form, or by any means electronic, mechanical,
+ * photocopying, or otherwise, without prior permission of the owner.
+ *
+ * © 2018 Feedzai, Strictly Confidential
+ */
+
+package com.feedzai.cosytest;
+
+
+import scala.Option;
+import scala.Predef;
+import scala.collection.JavaConverters;
+import scala.concurrent.duration.FiniteDuration;
+import scala.runtime.BoxedUnit;
+import scala.util.Try;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+
+public class DockerComposeJavaSetup {
```

Comment:
This file can be a Scala file, from what I can see. That would simplify code like `Predef.$conforms`.